Command/GenerateTokenCommand.php

```diff
@@ -44,9 +44,10 @@ protected function configure()
     {
         $this
             ->setName(static::$defaultName)
-            ->setDescription('Generates a JWT token')
-            ->addArgument('username', InputArgument::REQUIRED)
-            ->addOption('user-class', 'c', InputOption::VALUE_REQUIRED)
+            ->setDescription('Generates a JWT token with optional payload')
+            ->addArgument('username', InputArgument::REQUIRED, 'Username of user to be retreived from user provider')
+            ->addArgument('ttl', InputArgument::OPTIONAL, 'Ttl in seconds to be added to current time. If not provided, the ttl configured in the bundle will be used. Use 0 to generate token without exp', null)
```

Comment:
What about making it an option instead?

Reply:
I have no objection to that. Will amend accordingly.